packages/devtools/lib/src/ui/viewport_canvas.dart

```diff
@@ -184,11 +184,28 @@ class ViewportCanvas extends Object with SetStateMixin {
       // async so we risk flickering UI if we don't render with a buffer.
       rebuild(force: false);
     });
+
     if (_onTap != null) {
-      _content.onClick.listen((e) {
-        _onTap(_clientToGlobal(e.client));
+      _content.onClick.listen((e) => _onTap(_clientToGlobal(e.client)));
+
+      // We do not have a simple `onTouch` method to use, so we use
+      // `onTouchStart`, `onTouchMove`, and `onTouchEnd` to accomplish the same
+      // end as well as disambiguate between touch-drags and touches.
+      _content.onTouchStart.listen((e) {
+        final TouchEvent t = e;
+        // If there are multiple touches, always use the first.
+        activeTouch = t.touches.first;
+      });
+      _content.onTouchMove.listen((_) => wasDraggedByTouch = true);
```

Comment:
nit: prefer { to => as this isn't actually intended to return a value.}

Reply:
done